version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.1.0-SNAPSHOT"
+version in ThisBuild := "1.0.1-SNAPSHOT"
```

Comment:
Not a big deal, wondering if this was intentional, instead of `1.1.1` or `1.2.0`

Reply:
Yes, this is how the sbt-release plugin works with versions. The version committed to the repo is the one that will be released next, so in this case the [most recently released version](
https://github.com/Dwolla/sbt-dwolla-base/releases/tag/v1.0.0) is 1.0.0 and I want to release a bug fix.